services/web/server/src/simcore_service_webserver/folders_handlers.py

```diff
@@ -0,0 +1,100 @@
+import sqlalchemy as sa
+from aiohttp import web
+from servicelib.aiohttp.application_keys import APP_DB_ENGINE_KEY
+from sqlalchemy import and_
+
+from .db_models import folders
+from .login.decorators import RQT_USERID_KEY, login_required
+from .security_api import check_permission
+
+
+@login_required
+async def list_folders(request: web.Request):
+    await check_permission(request, "folder.crud.*")
+    uid, engine = request[RQT_USERID_KEY], request.app[APP_DB_ENGINE_KEY]
+    result = []
+    async with engine.acquire() as conn:
```

Comment:
Handler implementation should hide the details of the database under a "db repository". 
Same as what you did in the ``projects_folders_handlers`` with ``ProjectFolderDb``.
That will allow accessing that table from other handers (e.g. to aggregate) using a common interface